### ISSUE TITLE
Subjects Page:  interactive elements skipped on Firefox and Safari

### ISSUE THREAD

**Issue by ma8642** [internetarchive/openlibrary#5032]
Issue related to #4899 

### 2.ii.(a)
> Every time you encounter an interactive element 
> Make sure the element receives focus in a logical place on the page
> Here we want to check that we haven't skipped anything when going from one interactive element to another

## Observations:
1. :x: Only the search bar and search button receive focus.  Right after the search button, the next focused item is the browser reload button (Firefox).
2. :x: Only the search bar receives focus.  Right after the search button, the next focused item is the browser url field (Safari).
### Evidence / Screenshot
![subjects-page-interactive-elements](https://user-images.githubusercontent.com/13713037/114292399-547bcb80-9a5c-11eb-8c35-a376f0ad07f1.png)
Note:  All interactive elements receive focus on Chrome.

### Relevant url
`https://openlibrary.org/subjects`

### Steps to Reproduce

1. Go to https://openlibrary.org/subjects.
2. Once the page loads press tab repeatedly until you see focused element.
3. Notice the first focused element after the header is the searchbar.
4. Notice the second focused element after the header is the search button.

* Actual: :x: The only two focused elements are the search bar and the search button.
* Expected: Other interactive buttons should receive focus as well.  Some suggestions:
   * The edit button
   * the many subject links on the page
   * the revisions link in the "History" section at the bottom
   * the username link for the user who last edited.

### Details

- **Logged in (Y/N)**: N
- **Browser type/version:**
   - type:  Firefox / version: 86.0
   - type:  Safari / version: 14.0.3
- **Operating system:** MacOS
- **Environment (prod/dev/local)?** prod

### Proposal & Constraints

Check this page on Chrome to see how it should behave.

### Stakeholders

@bpmcneilly

**Comment by Yashs911** [comment]
1. Can't reproduce this error (Firefox).
![image](https://user-images.githubusercontent.com/64412143/114294342-98ea8400-9abb-11eb-98a5-f8a614ee21da.png)
But I believe the default focus outline is very thin and due to low color contrast, it is not visible.
2. Current Order of focus is `Edit>User>History` but it should be `User>History>Edit`
![image](https://user-images.githubusercontent.com/64412143/114294442-4d84a580-9abc-11eb-9639-7ecb76401536.png)

**Comment by jimchamp** [comment]
I am also unable to reproduce this in Firefox.  If it is no longer an issue when using Safari, this can be closed.
